environment_setup/ci_environment.yml

```diff
@@ -0,0 +1,26 @@
+name: mlopspython_ci
+
+dependencies:
+
+- r
+- r-essentials
+- numpy
+- pandas
+- scikit-learn
+
+- pip
+- pip:
+
+  # dependencies shared with diabetes_regression/scoring/scoring_dependencies.yml
+  # and/or diabetes_regression/training/training_dependencies.yml.
+  # If versions are specified, they must match exactly, or the docker build will fail.
+  - azureml-sdk==1.0.74
```

Comment:
This would be a nice opportunity to upgrade the aml sdk  to the latest and see if it works!

Reply:
Maybe let's also keep that for the next PR

Reply:
no worries if you don't want to push that change now